### DISCUSSION TITLE
Cannot sign in when backblaze account is from another oauth provider; also white screen to enter email

### DISCUSSION THREAD

**Discussion by alapins** [JonathanTreffler/backblaze-personal-wine-container#153] [Q&A]
Problem Description:
I use my google account to sign into backblaze.  After entering my email into the backblaze installer, it shows a login screen with the password label, but no password textbox, and the install now button is disabled (it doesn't look disabled, but it can't be clicked).

Solution:
Exit the installer.
From Start choose "run..."
click Browse
c:\Program Files (x86)\Internet Explorer\iexplore.exe

It will prompt that the gecko engine is not installed, but wine can install it automatically.
click "Install"

relaunch the backblaze installer, and the first screen now shows meaningful text in addition to the email field, and you can log into accounts using oauth providers.

Posting for others having this problem, and also perhaps the docker image can be updated so that wine installs the iexplore rendering engine?

**Comment by traktuner** [comment]
Hey,
currently I think your solution is the easiest one. I will look if gecko (mshtml) can easily be preinstalled to support 3rd party logins.
